turbonomic/operands/cert-manager.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: devops.turbonomic.io/v1alpha1
 kind: OperatorResourceMapping
 metadata:
-  name: cert-manager-orm
-  namespace: $namespace
+  name: cert-manager-config-orm
+  namespace: {{ placeholder_namespace }}
+  labels:
+    component: cpfs
```

Comment:
Although cert manager CR is cluster scope resource, I am wondering if this is the best practice to apply cert manager ORM in CPFS namespace? It should still work because cert manager CR is visible in CPFS namespace. Would it be better to have cert manager ORM deployed in cert manager namespace?

Reply:
Good point, will adjust so that it accepts cert-manager namespace and use ibm-cert-manager as default